napari/layers/image/image.py

```diff
@@ -93,12 +111,14 @@ class Image(Layer):
     """
 
     _colormaps = AVAILABLE_COLORMAPS
+    _max_tile_shape = 1600
```

Comment:
seems small?

Reply:
for a hardcoded value I mean.

Reply:
if the tiles are larger it can be more expensive to fetch them, and also at 1-1 resolution matching with the screen 1600 x 1600 fills our viewport, but maybe I'm missing something. Why would you expect it to be larger?

Reply:
Does it fill the viewport in a Retina display? What about a 4K monitor? Anyway I’m fine with it but I would just think it should be configurable...